.github/workflows/publish-release-artifact.yml

```diff
@@ -13,9 +13,9 @@ jobs:
     steps:
       - uses: actions/checkout@v3
       - name: Set up JDK 8
-        uses: actions/setup-java@v2
+        uses: actions/setup-java@v3.0.0
```

Comment:
https://github.com/jenkinsci/file-parameters-plugin/pull/90#discussion_r835918340 I _think_ also applicable here given https://github.com/jenkinsci/jenkins/blob/abd028bbb373a87708d60ad3d2e5b18b589c7e54/.github/workflows/publish-release-artifact.yml#L3-L5

Reply:
What is the purpose of reviewing a PR that has already been merged?

Reply:
Just informational, in case the author (or anyone else) thinks it would be appropriate to file a follow-up.

(In this case I did not see this PR until after it is was merged, due to a link from a plugin PR.)

Reply:
Have you considered doing a quick glance at the PRs with the `ready-for-merge` label every day or two to see if you have feedback for them? If the purpose of code review is to make the code of tangibly higher quality, then leaving feedback before a pull request is merged is far more likely to achieve this outcome. To be clear, there are many valid reasons for reviewing a PR after it has been merged, such as to notify authors of serious regressions, but in cases like this where there is no regression and where the suggestion is fairly trivial, it can come across as an example of "bring me a rock" style leadership.

Reply:
I am not familiar with that metaphor.

I do not normally attempt to follow all core PRs (or PRs in major plugins for that matter) since I simply lack the time for that sort of volume and there are already a number of diligent reviewers. This one I just stumbled across after the fact, via a link from a PR in a plugin I help maintain.